src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.cs

```diff
@@ -675,6 +675,12 @@ public static void ThrowObjectDisposedException_Utf8JsonWriter()
         {
```

Comment:
Could the two helpers just be merged into one?
```
 [DoesNotReturn]
public static void ThrowObjectDisposedException(string objectName)
{
      throw new ObjectDisposedException(objectName);
}
```

Reply:
We can, but it might make `CheckNotDisposed` non-inlineable due to increased IL size. Also, string literals as paramaters for throw-helpers usually lowered to additional calls (lazyStringLiteralInit), e.g. https://sharplab.io/#v2:C4LghgzgtgPgAgJgIwFgBQcDMACR2DC6A3utmdqeXEgGy4As2AKgKYTBIAUAlgHbDYwASkpkSacpOzcAZtk5hsAXiXZ6CERKnamACwBOAewDuAeQBGAKxYBjYABFuEAA6GILACYBRAB42WzsDchrwA+gBSECH2hjYArlAs/JxCANyi2AC+GTlaZNR0cIys7Ag8/IKakuLaZLLyiipqGhm1zAYmFtZ2ji5unr7+gcG8nABEkdGxCUnAY2kZ2XkUy1i4tAztRmZWtg5Oru7efgFBIZzUAAzYhrt2AHJgiVXkNW3YwB3G2Lws3117XqHAYnYbnW7dYCPZ7pZZLDJrAqbPTbAE9A79Y5DM5hSa8GLxRLJF5iVpST7bH5/bBo/Z9I6DU4jcZ4gkzfjzWGSJaZIA==

Reply:
Let's keep as-is in that case.